app/javascript/legacy/nonprofits/donate/info-step.js

```diff
@@ -106,7 +106,7 @@ function view(state) {
   , customFields(state.params$().custom_fields)
   , dedicationLink(state)
   , app.nonprofit.no_anon ? '' : anonField(state)
-  , h('fieldset.u-inlineBlock.u-marginTop--10', paymentMethodButtons(["card", "sepa"], state))
+  , h('fieldset.u-inlineBlock.u-marginTop--10.u-marginLeft--65', paymentMethodButtons(["card", "sepa"], state))
```

Comment:
Have you tried using something that sets `text-align: center` on this element? I think that would be the most ideal.

Reply:
We tried with solutions like that, but some problems occurred. Now that we have a better understanding of the structure of the project we will implement a solution of the type. Thanks for the feedback!

Reply:
@kayrocesar when you say problems occurred, could you describe them in more detail? It's good to have it documented and also so I can help you and you can help me.

Reply:
Hello @wwahammy, sorry for the delay in answering. 
We tried to use text-align but it didn't work before, because we were doing it in the wrong place. But we were able to align the buttons using it now, if you could take a look. Thanks a lot.